contracts/SpokePool.sol

```diff
@@ -753,19 +779,19 @@ abstract contract SpokePool is
      * _verifyUpdateV3DepositMessage() for more details about how this signature should be constructed.
      */
     function speedUpV3Deposit(
-        address depositor,
+        bytes32 depositor,
         uint32 depositId,
         uint256 updatedOutputAmount,
-        address updatedRecipient,
+        bytes32 updatedRecipient,
         bytes calldata updatedMessage,
         bytes calldata depositorSignature
     ) public override nonReentrant {
         _verifyUpdateV3DepositMessage(
-            depositor,
+            depositor.toAddress(),
```

Comment:
maybe also update _verifyUpdateV3DepositMessage interface so we don't need to translate to address here